packages/cape/src/methods.ts

```diff
@@ -360,6 +365,14 @@ export abstract class Methods {
   private async verifyAttestationDocument(message: string, checkDate?: Date): Promise<AttestationDocument> {
     const doc = parseAttestationDocument(message);
 
+    if (this.nonce != undefined) {
```

Comment:
We also throw an error if `this.nonce` is undefined since this should be a required step in validating the attestation document.

Reply:
We use this when requesting key and it won't have a match nonce unfortunately so it needs to be configurable.